src/sqlfluff/rules/jinja/JJ01.py

```diff
@@ -48,6 +48,9 @@ class Rule_JJ01(BaseRule):
     # sections.
     crawl_behaviour = SegmentSeekerCrawler({"raw"})
     targets_templated = True
+    # NOTE: This rule does return fixes, but when is_fix_compatible
+    # is set, it spirals.
+    # is_fix_compatible = True
```

Comment:
???

Reply:
I've made a separate issue for this: https://github.com/sqlfluff/sqlfluff/issues/4474.

I left the comment there in case someone else tries to do the same as what I tried.